pkg/cmd/internal/issues/issues.go

```diff
@@ -98,10 +98,13 @@ func trimIssueRequestBody(message string, usedCharacters int) string {
 
 // If the assignee would be the key in this map, assign to the value instead.
 // Helpful to avoid pinging former employees.
+// An "" value means that issues that would have gone to the key are left
+// unassigned.
 var oldFriendsMap = map[string]string{
```

Comment:
What about you don't do this this way, and instead
use HTTP requests to a (local) instance of the roacher directory - https://github.com/cockroachlabs/roachers

this has exactly the data you need.

Reply:
well maybe not exactly, but we could extend it in that way.